src/dmd/backend/cv8.d

```diff
@@ -133,121 +147,150 @@ void cv8_writename(Outbuffer *buf, const char* name, size_t len)
  * Input:
  *      filename        source file name
  */
-void cv8_initfile(const char *filename)
+void cv8_initfile(const(char)* filename)
 {
     //printf("cv8_initfile()\n");
 
     // Recycle buffers; much faster than delete/renew
 
     if (!F1_buf)
-        F1_buf = new Outbuffer(1024);
-    F1_buf->setsize(0);
+    {
+        __gshared Outbuffer f1buf;
```

Comment:
Why did you move away from `new` ? There's a lot of redundant code here.

Reply:
because `new` doesn't work in betterC, and it's never free'd anyway.